src/marketing/newsletter-subscribers.md

```diff
@@ -4,31 +4,66 @@ title: Managing Subscribers
 
 As a best practice you should manage your subscription list on a regular basis, and make sure to process any requests to unsubscribe. In some jurisdictions, it is required by law that requests to unsubscribe are processed within a specific period of time.
 
+You can easily manage your subscribers using a simple list of active subscriptions. When a customer submits an unsubscribe request, you can simply apply an unsubscribe action to one or more selected subscriptions.
+
+In single site setups with multiple store views, a customer account subscription can be associated with a specific store view.
+
+In multi-store and multi-site setups with a global [customer account scope]({% link customers/account-scope.md %}), a customer account can be subscribed to newsletters for multiple sites/stores. In this case, you may want to edit the customer account to manage a group of subscriptions or cancel a subscription for a specific site/store to honor a request.
+
 If you want to use a third-party service to send newsletters, you can export your subscription list as a CSV or XML file.
 
-## Cancel a Subscription
+## Manage subscriptions for a customer
+
+1. On the _Admin_ sidebar, go to **Customers** > **All Customers**.
+
+1. Find the customer in the grid and click **Edit** in the _Action_ column.
+
+1. Click **Newsletter** in the left panel.
+
+1. Modify the subscription(s) for the customer according to your site/store setup:
+
+   For a single site/single store setup, you can simply select or clear the **Subscribed to Newsletter** checkbox.
+
+   ![Single store customer newsletter subscription checkbox]({% link images/images/newsletter-customer-single-store.png %}){: .zoom}
+   _Customer Newsletter subscription - Single Store Setup_
+
+   For a single site/multi-store setup, you can select or clear the **Subscribed to Newsletter** checkbox and set **Subscribed on Store View** to the correct store view for the subscription.
+
+   ![Multi-store customer newsletter subscription checkbox and store view selector]({% link images/images/newsletter-customer-multi-store.png %}){: .zoom}
+   _Customer Newsletter subscription - Multi-Store Setup_
+
+   For a multi-site/multi-store setup with a global customer account scope, the page displays the subscription status for all sites. You can select or clear the **Subscribed** checkbox and/or change the **Store View** for the subscription.
```

Comment:
```suggestion
   For a multi-site/multi-store setup with a global [customer account scope]({% link customers/account-scope.md %}), the page displays the subscription status for all sites. You can select or clear the **Subscribed** checkbox and/or change the **Store View** for the subscription.
```

Reply:
We already linked above. I don't think we should keep linking to the same places throughout the topic. It cause "link fatique".